editMember.php

```diff
@@ -2,60 +2,64 @@
 <html lang="en">
 
 <head>
-  <link href="./styles/styles.css" rel="stylesheet" type="text/css" />
-  <link rel="icon" href="assets/icons/SiOverleaf.svg" type="image/icon" />
-  <title>GoToGro | Edit Member</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="./styles/styles.css" rel="stylesheet" type="text/css" />
+    <link rel="icon" href="assets/icons/SiOverleaf.svg" type="image/icon" />
+    <title>GoToGro | Edit Member</title>
 </head>
 
 <body>
-  <?php include_once("navbar.inc"); ?>
-
-  <main>
-    <form method="post" action="processMEMBER.php" novalidate="novalidate" class="editMemberform">
-
-      <div class="editFormInput">
-        <h3>Edit Member Form</h3>
-      </div>
-
-      <div class="editFormInput">
-        <label for="firstname" id="fName">First Name</label>
-        <input type="text" pattern="[a-zA-Z]{2,20}" id="firstname" name="firstname" required />
-      </div>
-
-      <div class="editFormInput">
-        <label for="lastname" id="lName">Last Name</label>
-        <input type="text" pattern="[a-zA-Z]{2,20}" id="lastname" name="lastname" required />
-      </div>
-
-      <div class="editFormInput">
-        <label for="DateofBirth" id="dob">Date of Birth</label>
-        <input type="date" pattern="\d{1,2}\/\d{1,2}\/\d{4}" name="dateofbirth" id="dateofbirth" required />
-      </div>
-
-      <div class="editFormInput">
-        <label for="email" id="em">Email</label>
-        <input type="email" id="email" name="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required />
-      </div>
-
-      <div class="editFormInput">
-        <label for="phonenum" id="mobilenum">Mobile Number (Optional)</label>
-        <input type="tel" name="phonenum" id="phonenum" maxlength="12" pattern="[\d\s]{8,12}" placeholder="For eg. 0400 000 000" required />
-      </div>
-
-
-
-
-      <div class="button">
```

Comment:
major change in editMember.php is the class name for the button.. the rest your changes can apply